spec/ttt/game_rules_spec.clj

```diff
@@ -0,0 +1,53 @@
+(ns ttt.game-rules-spec
+  (:require [speclj.core :refer [describe context it should=]]
+            [ttt.game-rules :as rules]))
+
+
+(describe "Game Rules"
+  (context "#win?"
+    (it "returns false if there is no winner"
+      (should= false (rules/win? ["X" "X"  "O"
+                                  "O" "O"  "X"
+                                   6   7   "X"])))
+    (it "checks if the game is won for a row of X's"
+      (should= true (rules/win? ["X" "X" "X"
+                                 "O"  4  "O"
+                                  5   6   7])))
+    (it "checks if the game is won for a row of O's"
+      (should= true (rules/win? ["X"  1  "X"
+                                 "O" "O" "O"
+                                  5   6   7])))
+    (it "checks if the game is won for a column of X's"
+      (should= true (rules/win? ["X" "O" "O"
+                                 "X"  4  "O"
+                                 "X"  6   7])))
+    (it "checks if the game is won for a column of O's"
```

Comment:
I think rather than having 2 tests for the column of different markers, we should test for each of the three columns. I'm not sure if everyone does this but I like it mix up the parameters like you did with the O's and X's in similar tests. So, I think you can just keep these two tests and write one more for the 3rd column! (same thing for row)